docs/guides/server/hostname.adoc

```diff
@@ -40,7 +40,7 @@ Most of the time, it should be enough to set the `hostname` option in order to c
 When using the `hostname` option the server is going to resolve the HTTP scheme, port, and path, automatically so that:
 
 * `https` scheme is used unless you set `hostname-strict-https=false`
-* Use the standard HTTP ports (e.g.: `80` and `443`) if a `proxy` is set or use the port you set to the `hostname-port` option
+* if `proxy` option is set, the proxy is expected to use the default  ports (e.g.: 80 and 443), in case the proxy uses a different port, it needs to be specified via the `hostname-port` configuration option
```

Comment:
```suggestion
* if the `proxy` option is set, the proxy will use the default ports (i.e.: 80 and 443).  If the proxy uses a different port, it needs to be specified via the `hostname-port` configuration option
```

Reply:
@shawkins if I may ask, what's the rule of thumb when it comes to using e.g. vs. i.e.? There are still multiple places in within this guide, where e.g. is used. Should I fix it everywhere?

Reply:
> @shawkins if I may ask, what's the rule of thumb when it comes to using e.g. vs. i.e.? There are still multiple places in within this guide, where e.g. is used. Should I fix it everywhere?
> 
> Up to this moment I've thought we use e.g. when we list examples right after it.

The usage is roughly:
- e.g. lists one or more examples out of a larger possible set.
- i.e. lists all possible values - typically a small number

Reply:
Got it, thanks for the clarification!